modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientCodegen.java

```diff
@@ -624,6 +624,14 @@ public String toDefaultValue(Schema p) {
                 else
                     return "True";
             }
+            // include fallback to example, default defined as server only
+            // example is not defined as server only
+            if (p.getExample() != null) {
+                if (p.getExample().toString().equalsIgnoreCase("false"))
```

Comment:
What about using `Boolean.valueOf` instead of using `equalsIgnoreCase`?

Reply:
Boolean.valueOf(string) will always return True or False, 
so Boolean.valueOf("5") would return False. 
When I pass in "5" as a boolean I should return null. Code updated use equalsIgnoreCase to check for "true" and "false" otherwise return null.

Reply:
> When I pass in "5" as a boolean the code should return null

If someone puts down 5 as the example value and assume it will be treated false in boolean, it's the user's issue (not something we need to handle) and I don't think it should return null (clearly there's a value defined)

We've seen spec using 1 or 0 to as example value for boolean type (not entirely correct as they should be using "true" or "false" instead) but still we want to able to handle these cases correctly as some languages (e..g Perl) treat 1 as true.

Reply:
Coercing it to a boolean gives the user back some boolean which has worth. I will change it to Boolean.valueOf. This behavior may surprise python users though because they will be using a python client with java casting of string truthiness.
"1" -> True in Python, false in Java
"0" -> True in Python, false in Java